lib/course_planner/tasks.ex

```diff
@@ -47,22 +47,32 @@ defmodule CoursePlanner.Tasks do
     end
   end
 
-  def get_user_tasks(id) do
-    Repo.all(from t in Task, where: t.user_id == ^id, preload: [:user])
+  def get_for_user(id, now) do
+    Repo.all(from t in Task, where: t.user_id == ^id and t.finish_time > ^now, preload: [:user])
```

Comment:
I think it makes sense to split in the template so we would have 3 tables: available, future and past tasks